tests/interfaces/test_tf.py

```diff
@@ -666,7 +666,9 @@ def div(a, b):
             grad = tape.gradient(y, [a, b])
 
         assert grad[0].numpy() == 1 / b.numpy()
-        assert np.allclose(grad[1].numpy(), -a.numpy() / b.numpy() ** 2, atol=tol, rtol=0)
+        res = grad[1].numpy()
+        exp = -a.numpy() / b.numpy() ** 2
+        assert np.allclose(res, exp, atol=tf_tol, rtol=0)
```

Comment:
Or is this the only test? Weird :thinking:

Reply:
This one is the only one that failed

Reply:
might be worth checking if this now works with `tol` instead of `tf_tol`